src/components/AdditionalInfoPage/AdditionalInfoPage.js

```diff
@@ -125,6 +127,7 @@ const AdditionalInfoPage = () => (
         <h1
           id={ELEMENT_IDS.CONTENT_CREDIT}
           className={style.title}
+
```

Comment:
This is adding new blank lines, there should be no changes in this file

Reply:
Ah yes, that was my find and replace, whoops. We should make the linter pick this up.